index.html

```diff
@@ -1011,6 +1011,33 @@ <h4>Exchange Examples</h4>
     <h2>Privacy Considerations</h2>
     <p>
     </p>
+
+    <section>
+      <h3>Phone Home Considered Harmful</h3>
+
+      <p>
+It is considered a bad practice for a <a>verifier</a> to contact an
+<a>issuer</a> about a specific <a>verifiable credential</a>. This 
+practice is known as "phoning home" and can result in a mismatch 
+in privacy expectations between <a>holders</a>, <a>issuers</a>,
+<a>verifiers</a>, etc., as it allows <a>issuers</a> to correlate
+<a>holders</a> with <a>verifiable credentials</a> without their
+knowledge.
+      </p>
+
+      <p>
+Sometimes an <a>issuer</a> can be contacted to get status information in
+a privacy-respecting manner, such as through a status list that provides 
+herd privacy. This is ok as long as the <a>issuer</a> is not able to
+correlate a single individual based on the retrieval of the status list.
+      </p>
+
+      <p>
+<a>Verifiers</a> are urged to not "phone home" in ways that will create 
+privacy violations for any specific individual(s).
```

Comment:
I imagine this could apply to organizations as well, so perhaps we just strike the latter part.

```suggestion
privacy violations.
```